anitya/lib/exceptions.py

```diff
@@ -102,3 +102,22 @@ def __str__(self):
             return 'Invalid version "{v}": {e}'.format(v=self.version, e=str(self.exception))
         else:
             return 'Invalid version "{v}"'.format(v=self.version)
+
+
+class RateLimitException(AnityaException):
+    """
+    Raised when the rate limit for requests is reached.
+
+    Args:
+        reset_time (str): Time when limit will be reseted (UTC time encoded in ISO-8601).
+    """
+
+    def __init__(self, reset_time):
+        self._reset_time = reset_time
+
+    @property
```

Comment:
I recommend just calling ``self._reset_time`` ``self.reset_time`` and not making a property. Properties are handy if you need to do some sort of transformation or you want your internal representation to differ from the public API in some way, but here it's the same.